x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/total_feature_importance_summary/feature_importance_summary.tsx

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useCallback, useMemo } from 'react';
+import {
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIconTip,
+  EuiPanel,
+  EuiSpacer,
+  EuiTitle,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  Chart,
+  Settings,
+  Axis,
+  ScaleType,
+  Position,
+  BarSeries,
+  RecursivePartial,
+  AxisStyle,
+  PartialTheme,
+  BarSeriesSpec,
+} from '@elastic/charts';
+import { i18n } from '@kbn/i18n';
+import euiVars from '@elastic/eui/dist/eui_theme_light.json';
+import {
+  TotalFeatureImportance,
+  isClassificationTotalFeatureImportance,
+  isRegressionTotalFeatureImportance,
+  RegressionTotalFeatureImportance,
+  ClassificationTotalFeatureImportance,
+} from '../../../../../../../common/types/inference';
+import { useMlKibana } from '../../../../../contexts/kibana';
+const { euiColorMediumShade } = euiVars;
+const axisColor = euiColorMediumShade;
+
+const axes: RecursivePartial<AxisStyle> = {
+  axisLine: {
+    stroke: axisColor,
+  },
+  tickLabel: {
+    fontSize: 12,
+    fill: axisColor,
+  },
+  tickLine: {
+    stroke: axisColor,
+  },
+  gridLine: {
+    horizontal: {
+      dash: [1, 2],
+    },
+    vertical: {
+      strokeWidth: 0,
+    },
+  },
+};
+const theme: PartialTheme = {
+  axes,
+};
+
+export interface FeatureImportanceSummaryPanelProps {
+  totalFeatureImportance: TotalFeatureImportance[];
+}
+
+const tooltipContent = i18n.translate(
+  'xpack.ml.dataframe.analytics.exploration.featureImportanceSummaryTooltipContent',
+  {
+    defaultMessage:
+      'Shows to what degree a given feature of a data point contributes to the prediction. The magnitude of feature importance shows how significantly the feature affects the prediction both locally (for a given data point) or generally (for the whole data set).',
+  }
+);
+
+export const FeatureImportanceSummaryPanel: FC<FeatureImportanceSummaryPanelProps> = ({
+  totalFeatureImportance,
+}) => {
+  const {
+    services: { docLinks },
+  } = useMlKibana();
+
+  const [plotData, barSeriesSpec, showLegend, chartHeight] = useMemo(() => {
```

Comment:
For binary classification jobs, is it possible to edit the 'bars' label shown in the tooltip? If so, maybe a label like 'magnitude' might be better?

![image](https://user-images.githubusercontent.com/7405507/94416069-475d5600-0176-11eb-9ee3-dfc6aa11a83f.png)

Reply:
Updated here https://github.com/elastic/kibana/pull/78238/commits/6a3c0d8eb62eab743b4ddb334446f7eb475ec7a8